src/features/user/TreeMapper/Import/components/PlantingLocation.tsx

```diff
@@ -22,6 +19,10 @@ import tj from '@mapbox/togeojson';
 import gjv from 'geojson-validation';
 import flatten from 'geojson-flatten';
 
+import MuiDatePicker from '@mui/lab/MobileDatePicker';
```

Comment:
Color of selected date needs to be adjusted, and font color (when date is selected) should be white.

Reply:
https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1466#discussion_r866510455

Reply:
Issue created #1486